core/src/main/java/bisq/core/trade/statistics/TradeStatistics2.java

```diff
@@ -217,7 +218,7 @@ public static TradeStatistics2 fromProto(protobuf.TradeStatistics2 proto) {
                 proto.getTradePrice(),
                 proto.getTradeAmount(),
                 proto.getTradeDate(),
-                null, // We don't want to expose this anymore
+                ProtoUtil.stringOrNullFromProto(proto.getDepositTxId()),
```

Comment:
This would leave all new trades from older clients still exposed. That's perhaps the safest way to do it. Was hoping it would be possible to keep setting it to null for new version and just not serve the old version anything. Not sure if that would break things though.

Reply:
Yes, you only get the additionally privacy if you update or if you trade with a peer that has already updated. I had to do it like that to prevent the unnecessary load from old clients.